### DISCUSSION TITLE
cursor automaticaly in "Search in the zeb..."

### DISCUSSION THREAD

**Discussion by JamesDAdams** [bluecaret/carettab#354] [Ideas and Feature Requests]
Hi,

I use my Brave in fullscreen and vertical tabs, when i open a new tab i whant to have my cursor on "Search in the zeb..." for directly write.

Thanks

**Comment by bluecaret** [comment]
Thank you for the suggestion. Unfortunately, this is a restriction by the browser, it won't let the extension automatically focus on the search bar and I have not found a way around this yet.

**Reply by JamesDAdams** [reply]
Ok, thanks for the reponses verry fast :)
